ambuda/admin.py

```diff
@@ -21,7 +21,7 @@ def index(self):
 
 class BaseView(sqla.ModelView):
     def is_accessible(self):
-        return current_user.is_admin
+        return q.is_user_admin(current_user.id)
```

Comment:
The API feels a little boilerplate-y. What if we keep the `is_admin` property and do the query in `auth.py` ?